engines/director/resource.cpp

```diff
@@ -312,6 +313,12 @@ Archive *DirectorEngine::loadEXE(const Common::Path &movie) {
 		for (uint i = 0; i < versions.size(); i++) {
 			Common::WinResources::VersionInfo *info = exe->getVersionResource(versions[i]);
 
+			Common::String gameName = info->hash["FileDescription"];
```

Comment:
Do you really need to create the variables?
You could directly use the formatting capabilities of debugC to format the version.